gr-audio/lib/alsa/alsa_sink.cc

```diff
@@ -282,8 +283,6 @@ int alsa_sink::work(int noutput_items,
                     gr_vector_const_void_star& input_items,
                     gr_vector_void_star& output_items)
 {
-    assert((noutput_items % d_period_size) == 0);
```

Comment:
I think this change is OK, but in general, I'm OK with people asserting things that "should be like that". Who knows in which context code gets run in the future.